### ISSUE TITLE
Training loss

### ISSUE THREAD

**Issue by DevKretov** [google-research/electra#26]
Hello,

I was wondering whether it is possible to add some loss metrics to the training cycle? The only thing I see during training Electra model is

`1275000/3000000 = 42.5%, SPS: 3.1, ELAP: 9:24:02, ETA: 6 days, 11:55:19`

which tells nothing about how good is it. I'm trying to add some code to the estimator, but it seems to me that it could be much easier to show all the metrics in order to see how successful the model is at this stage.

**Comment by 008karan** [comment]
You can always use tensorboard though!

**Comment by chriskhanhtran** [comment]
@008karan Hi Karan. I am not a TF user. Can you please instruct me how to use tensorboard in this case? I see an tfevents file in the model directory but it seems not to be written for tensorboard. The script used `tf.estimator.tpu.TPUEstimator` to train the model so I don't know how to extract the loss. Thank you very much in advance!

**Comment by 008karan** [comment]
I have trained my model on GPU and using tensorboard is similar here. You will find events.out files in your checkpoint folder. Just run tensorboard on it.

**Comment by chriskhanhtran** [comment]
@008karan Thank you Karan! It works for me now after I use tensorboard==1.15.0. Do you know how the author can continuously get the evaluation metrics as in [this thread](https://github.com/google-research/electra/issues/3)? I can only get the evaluation metrics at the end of my training progress.

**Comment by 008karan** [comment]
In tensorboard you get loss and learning rate here I think you can add whatever you want in logs to see them on tensorboard!

**Comment by mrm8488** [comment]
When I trained Electra small on my Spanish corpus the kids was shown if trained on GPU. Then I got access to TFRC and trained in using its TPUs pod and loos it is not shown. Of course, I can get it from Tensorboard events but would be great to log it by default when running on TPU.

**Comment by nemaniarjun** [comment]
You can set the tensorflow log level to info and it will be much more verbose including printing the loss.
